htdocs/loan/payment/payment.php

```diff
@@ -152,9 +152,9 @@
 			$payment->amount_capital	= $pay_amount_capital;
 			$payment->amount_insurance	= $pay_amount_insurance;
 			$payment->amount_interest	= $pay_amount_interest;
-			$payment->fk_bank           = GETPOST('accountid', 'int');
+			$payment->fk_bank           = GETPOSTINT('accountid');
 			$payment->paymenttype       = GETPOST('paymenttype', 'int');
-			$payment->num_payment		= GETPOST('num_payment');
+			$payment->num_payment		= GETPOSTINT('num_payment');
```

Comment:
num_payment is a reference of payment. It can contains chars. For example
12345-chq1
So should remain GETPOST(. , alphanohtml)